### ISSUE TITLE
SLKTextViewController.h not found.

### ISSUE THREAD

**Issue by AdamBCo** [slackhq/SlackTextViewController#286]
For some reason the header files are not being linked in the update when installing the SlackTextViewController with Cocoapods.

**Comment by dzenbot** [comment]
Which Cocoapods version is that? Which version of the library are you installing?

**Comment by AdamBCo** [comment]
0.39.0

**Comment by Gman9855** [comment]
Having the same problem.  Even tried manually adding the library, same error.  SLKTextView+SLKAdditions.h not found when it's being imported from SLKTextViewController.

**Comment by AdamBCo** [comment]
I tried adding the static library as well, and ran into the same issue.

I also cleaned the project, restarted Xcode, restarted my Macbook Pro, and did a long list of other recommendations as well.

**Comment by AdamBCo** [comment]
Occasionally Xcode will autocomplete the following, but it won't show any of the files referenced inside;

#import <SlackTextViewController/>

**Comment by AdamBCo** [comment]
@Gman9855 I got everything to work again, so hopefully this helps.

For some reason, my targets User Header Search Paths were empty.

I had to paste the following inside to get it to work: $(BUILT_PRODUCTS_DIR)

**Comment by Gman9855** [comment]
Didn't work for me unfortunately.  Still getting 'file not found' when i do '#import <SlackTextViewController/SLKTextViewController.h>' or any of the other variations i've tried :(

**Comment by neil-wu** [comment]
I ran into the same issue.

**Comment by dzenbot** [comment]
Nothing changed on the library lately regarding the `podspec`.
I'm still on Cocoapods `0.38.2`. I should try `0.39.0` and see if I can repro.

**Comment by Soryu** [comment]
It’s because of the way slack source files are split into `Additions` and `Classes` and CocoaPods 0.39 has changed something with regard to the header search paths. It can be fixed by manually adding

    "${PODS_ROOT}/Headers/Public/SlackTextViewController/Additions"
    "${PODS_ROOT}/Headers/Public/SlackTextViewController/Classes"

to the target's `Header Search Path` build setting. Or by the Slack team restructuring their source files. ;)

**Comment by Gman9855** [comment]
When i do that i get this...

![screen shot 2015-10-15 at 9 55 42 am](https://cloud.githubusercontent.com/assets/5007053/10521150/07760870-7323-11e5-88dc-a0b9a900437f.png)

**Comment by dzenbot** [comment]
After some testing, it seems that this should be now fixed with [release 1.7.1](https://github.com/slackhq/SlackTextViewController/releases/tag/v1.7.1). Could anyone confirm please?

**Comment by vampirewalk** [comment]
It works for me with release 1.7.1. Thanks.

**Comment by neil-wu** [comment]
Also works for me with release 1.7.1. 
Thanks.

**Comment by dzenbot** [comment]
✌️

**Comment by A1iAshoor** [comment]
w/ swift:

In Bridging-header.h:

     #import <SlackTextViewController/SLKTextViewController.h>

Then run in Terminal:

     run pod install

Then Import in your controllers:
 
     import SlackTextViewController

**Comment by blangslet** [comment]
@A1iAshoor I'm getting ```Use of unresolved identifier 'SLKMessageViewLayout'``` and ```Use of unresolved identifier 'SLKMessageViewCell'``` when using the swift version in a current project. Are you seeing anything similar, or know how I could troubleshoot these errors?

**Comment by blangslet** [comment]
it looks like a Bridge-Header.h file of: 
```#import "SlackTextViewController/SLKTextViewController.h"```
```#import "SLKMessageViewLayout.h"```
```#import "SLKMessageViewCell.h"```
builds successfully, the view seems a bit strange though. Will dig in more tomorrow.

**Comment by dzenbot** [comment]
@blangslet `SLKMessageViewLayout` isn't part of the library. It's a class from the sample project.
Are you trying to reuse that class?

**Comment by blangslet** [comment]
@dzenbot Yep, I ended up just bringing over the objective-c classes from the sample project and was able to import them in the bridge-header. Thanks for your work on this, pretty awesome project! :)

**Comment by dzenbot** [comment]
💪😎 Thanks!
The `SLKMessageViewLayout` is a work-in-progress thing that I never continued working on.
If you do end up using it and improving it, please send a PR to review it and hopefully include it in the sample project.

**Comment by blangslet** [comment]
@dzenbot definitely will do! Going to use this in a night/weekend project, hopefully integrating media sharing won't be too difficult. I'm also considering trying to implement conversation threading, but that's a whole different ballgame. I'll send in PRs for any fun things that come out of the experimentation. 💁

**Comment by AndySoHotRightNow** [comment]
Just an observation. I installed SlackTextViewController via Cocoapods. When I first tried to subclass SlackTextViewController, I selected 'SLKTextViewController' as the parent class. The .h file that got created automatically included the following import statement: `#import <SlackTextViewController/SlackTextViewController.h>` and I got the error `'SlackTextViewController/SlackTextViewController.h' file not found`. It was sufficient for me to change the import statement to `#import <SlackTextViewController/SLKTextViewController.h>` to get it working.
